src/function/table/system/test_all_types.cpp

```diff
@@ -204,13 +204,14 @@ vector<TestType> TestAllTypesFun::GetTestTypes(bool use_large_enum) {
 	auto max_map_value = Value::MAP(ListType::GetChildType(map_type), map_values);
 	result.emplace_back(map_type, "map", std::move(min_map_value), std::move(max_map_value));
 
+#ifndef DUCKDB_WASM
```

Comment:
Is this necessary? Can't we add the `EXCLUDE` on the DuckDB-WASM side?

Reply:
Makes sense, will do that.